app/jobs/sync/zendesk/ticket_comments_to_notes.rb

```diff
@@ -0,0 +1,55 @@
+class Sync::Zendesk::TicketCommentsToNotes
+  include Sidekiq::Worker
+
+  # https://developer.zendesk.com/rest_api/docs/support/ticket_comments#list-comments
+  def perform(zendesk_ticket_id, page_number = 0)
+    Rails.logger.info "Starting zendesk ticket #{zendesk_ticket_id}"
+    require 'zendesk_api'
+    client = ZendeskAPI::Client.new do |config|
```

Comment:
I'd prefer this be abstracted out so we don't have duplicate code, but it's fine for now.